### PR TITLE
Home screen

### PR DESCRIPTION
**Pull request by natjoe4**
# Summary
Added a basic homescreen with links and information on how to use the app

## New behavior
Now the app will load a homescreen on startup

## Code changes
Added `Homescreen.tsx` with links and info on app usage

# Testing guidance
Make sure I didn't miss any important links or info! Barebones now, but we'll add to this as the app develops.

### CODE REVIEW

File: frontend/pages/index.tsx [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
 
 export default function Home() {
-  return (
-    <>
-      <Center style={{ alignItems: 'center', paddingTop: '80px' }}>
-        <div>Coming soon!</div>
-      </Center>
-    </>
-  );
+  return <Homepage />;
```
Comment (lmd59) [inline frontend/pages/index.tsx:5]: Probably a matter of opinion, but is it a bit unnecessary to have the homepage be a separate page that gets pulled into the index.tsx? Like homepage is index and index is homepage, so it kind of feels like 2 names for the same thing.

File: frontend/pages/Homepage.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      </Title>
+      <Text>
+        The backend Measure Repository Service server must be running on port 3000 in order for this application to pull
+        data from the server. To start the backend Measure Repository Service server on port 3000, run &quot;npm run
+        start:all&quot; or &quot;npm run start:backend&quot; from the root of the repository.
+      </Text>
+      <Text>More functionality coming soon!</Text>
+      <Divider my="sm" variant="dotted" />
+      <Title order={2} underline>
+        Authoring Functionality:
```
Comment (lmd59) [inline frontend/pages/Homepage.tsx:33]: Should we add anything about our intentions for the authoring functionality at this juncture? Maybe at least also link to the spec here, too, so there's a connection to what authoring means in this context?

File: frontend/pages/Homepage.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          </Anchor>
+        </List.Item>
+        <List.Item>
+          <Anchor href="https://github.com/projecttacoma/measure-repository">
+            ProjectTacoma Measure Repository Github
+          </Anchor>
+        </List.Item>
+        <List.Item>
+          <Anchor href="https://github.com/projecttacoma/measure-repository">
+            ProjectTacoma Measure Repository Backend README
```
Comment (lmd59) [inline frontend/pages/Homepage.tsx:58]:
```suggestion
            ProjectTacoma Measure Repository README
```
The README also discusses the front end, so I think we can just say "README"

Comment (natjoe4) [reply frontend/pages/Homepage.tsx:58]: Whoop! Linked to the wrong readme. Is this better?

File: frontend/pages/index.tsx (hunk 1 above)
Comment (mgramigna) [reply frontend/pages/index.tsx:5]: Yeah I would just put the contents of the `homepage` component inside `index.tsx` directly.

Comment (mgramigna) [reply frontend/pages/index.tsx]: Also putting `Homepage.tsx` in the `pages` directory will create a route to `localhost:3000/Homepage` for the content, which is not what we want. If you were to separate out the content of a page into its own component, use a `components` directory or something. `pages` should be reserved for things that will be directly routed to

File: frontend/pages/index.tsx [hunk 4]
```diff
@@ -1,11 +1,72 @@
...
+          </Anchor>
+        </List.Item>
+        <List.Item>
+          <Anchor href="https://github.com/projecttacoma/measure-repository">
+            ProjectTacoma Measure Repository Github
+          </Anchor>
+        </List.Item>
+        <List.Item>
+          <Anchor href="https://github.com/projecttacoma/measure-repository/blob/main/backend/README.md">
+            ProjectTacoma Measure Repository backend README
```
Comment (lmd59) [inline frontend/pages/index.tsx:66]:
```suggestion
            ProjectTacoma Measure Repository Backend README
```

**Review by lmd59 — COMMENTED** [review]
Maybe we can add a link to the frontend README at the end of the list as well, if you think that's appropriate:
`<List.Item>
        <Anchor href="https://github.com/projecttacoma/measure-repository/blob/main/frontend/README.md">
          ProjectTacoma Measure Repository Frontend README
        </Anchor>
      </List.Item>`

**Review by lmd59 — APPROVED** [review]
Looks good to me!

File: frontend/pages/index.tsx [hunk 5]
```diff
@@ -1,11 +1,77 @@
...
+        data from the server. To start the backend Measure Repository Service server on port 3000, run &quot;npm run
+        start:all&quot; or &quot;npm run start:backend&quot; from the root of the repository.
+      </Text>
+      <Text>More functionality coming soon!</Text>
+      <Divider my="sm" variant="dotted" />
+      <Title order={2} underline>
+        Authoring Functionality:
+      </Title>
+      <Text>
+        This application will provide a FHIR Measure authoring environment as outlined in the{' '}
```
Comment (elsaperelli) [inline frontend/pages/index.tsx:36]:
```suggestion
        This application will provide a FHIR Measure and Library authoring environment as outlined in the{' '}
```
? Or is the original correct

**Review by elsaperelli — CHANGES_REQUESTED** [review]
Nice! Just one small inline comment. Also, should we add one sentence about this also being useful for simply browsing through a measure repository?

File: frontend/pages/index.tsx [hunk 6]
```diff
@@ -1,11 +1,93 @@
...
+      <Text>More functionality coming soon!</Text>
+      <Divider my="sm" variant="dotted" />
+      <Title order={2} underline style={{ marginBottom: '8px' }}>
+        Capabilities
+      </Title>
+      <Title order={3} underline>
+        Resource Read and Search
+      </Title>
+      <Text style={{ marginBottom: '8px' }}>
+        This application provides and interface for Measure Repository Service read and search operations. Users can
```
Comment (elsaperelli) [inline frontend/pages/index.tsx:39]:
```suggestion
        This application provides an interface for Measure Repository Service read and search operations. Users can
```
😄

File: frontend/pages/index.tsx [hunk 7]
```diff
@@ -1,11 +1,93 @@
...
+        view the JSON content of resources on the server by clicking the desired resource resource type in the
```
Comment (elsaperelli) [inline frontend/pages/index.tsx:40]:
```suggestion
        view the JSON content of resources on the server by clicking the desired resource type in the
```
One too many resources?

**Review by elsaperelli — CHANGES_REQUESTED** [review]

**Review by elsaperelli — APPROVED** [review]

**Review by lmd59 — APPROVED** [review]
Super-approve!

**Comment by mgramigna** [comment]
image

The double heading feels redundant to me. Can we unify this somehow? What about just "Measure Repository" as a top banner or something

**Pull request merged**